### DISCUSSION TITLE
A button for resetting offset animation

### DISCUSSION THREAD

**Discussion by Chrommanito** [mrachinskiy/commotion#30] [Ideas]
After I offset my animation, it took me sometime and confusion to revert the changes from the offset animation. Eventually I figured out that you can set your offset to 0 and re-apply your offset animation. It would be convenient to beginners and more user-friendly if there's a button to do this automatically without manually changing your offset.

Thank you

**Comment by mrachinskiy** [comment]
What is the practical use case for that button, and how often it will be used?

For new users better educational material would be much more useful, than additional button.

**Reply by Chrommanito** [reply]
For one, I couldn't find this problem on google and no one's discussing it. It might be easy to figure out, but stuff that is easy to figure out can also be easily found on google.

So whether it's a single thread on github or stackexchange, an FAQ, or a feature inside the add-on.

**Reply by mrachinskiy** [reply]
You can create question on stackexchange and answer it yourself, that's what I did when I could't find answers in Blender documentation.

As I see it, separate button will be useless for everyone, including you, since you already know how to align animation start to a single frame.
